data/sql/updates/pending_db_world/rev_1735425042597130000.sql

```diff
@@ -0,0 +1,2 @@
+-- Remove HARD_RESET flag while retaining IMMUNITY_KNOCKBACK
+UPDATE `creature_template` SET `flags_extra` = `flags_extra` &~ (2147483648) WHERE `entry` = 36658;
```

Comment:
He should despawn on evade, so this is wrong

Reply:
It's definitely a hack on the current hack that exists, but seems to at least fix (at the moment) the boss "reseting" if players die at him. Think the logic is that because line 100 pInstance->SetData(DATA_TYRANNUS, NOT_STARTED);  in reset() sets to not even started (soft reset ish?); the HARD_RESET must not happen. Perhaps you have more thoughts @Talamorts1

Reply:
> He should despawn on evade, so this is wrong

where is your evidence of this? this Flag was applied in a previous "Fix",

if this is the case then you need rework your instance script to spawn npc if event is not done and creature is not in instance.